packages/react-ui/src/hooks/useCompoundComponents/helpers/getDefaultDataRender.tsx

```diff
@@ -48,7 +48,7 @@ export function getDefaultDataRender<
     }
 
     if (type === "date" || type === "dateonly" || type === "datetime") {
-      return <Date value={value} dateOnly={type === "dateonly"} />
+      return <Date value={value} step={control.step} />
```

Comment:
There shouldn't be a "dateonly" type or a "datetime" type.  Is this for legacy / backward compat support?

Reply:
@roy-coder i think the change you've made on line 51 is correct

Justin is referring to line 50. In the schema attributes, the `control.type` that is set is incorrect

```
datetime() // orm.type = "date" (I think) and control.type = "datetime"
dateonly() // orm.type = "dateonly" (I think) and control.type = "dateonly"
```

since control type is only used by the frontend and it's used to figure out which inputs should be rendered, both `datetime()` and `dateonly()` attributes should return `control.type` and then `step` is used to determine the input (datepicker vs datetimepicker):

```
if (type === "date") {
  return <Date value={value} step={control.step} />
}
```